js/src/settings.js

```diff
@@ -215,7 +215,21 @@
     'lockController' : {
       'lockProfile' : 'lazyZoom',
       'notifyMaxMin' : true
-    }
+    },
+	
+	//Default settings for the sanitizeHTML function
+	"sanitizeHtmlSettings": {
```

Comment:
It looks like the rest of the file is 2 space soft indentions. Could you standardize with that? That will soon be part of the Mirador CI build when https://github.com/ProjectMirador/mirador/pull/1337 is merged. There seems to be a concensus on that approach.